tests/unit/10_apps.bats

```diff
@@ -90,3 +90,36 @@ teardown () {
   echo "status: "$status
   assert_success
 }
+
+@test "(apps) apps:clone" {
+  deploy_app
+  run bash -c "dokku apps:clone $TEST_APP great-test-name"
+  echo "output: "$output
+  echo "status: "$status
+  assert_success
+  run bash -c "dokku apps | grep $TEST_APP"
+  echo "output: "$output
+  echo "status: "$status
+  assert_success
+  run bash -c "curl --silent --write-out '%{http_code}\n' `dokku url great-test-name` | grep 404"
+  echo "output: "$output
+  echo "status: "$status
+  assert_output ""
+  run bash -c "dokku --force apps:destroy great-test-name"
+  echo "output: "$output
+  echo "status: "$status
+  assert_success
+
+  run bash -c "dokku apps:clone --skip-deploy $TEST_APP great-test-name"
+  echo "output: "$output
+  echo "status: "$status
+  assert_success
+  run bash -c "curl --silent --write-out '%{http_code}\n' `dokku url great-test-name` | grep 404"
```

Comment:
Not clear as to why both `curl` tests should return 404. Seems like the one without `--skip-deploy` should return actual app output.